src/installer.ts

```diff
@@ -170,20 +196,68 @@ export async function getNode(
   core.addPath(toolPath);
 }
 
+function isLtsAlias(versionSpec: string): boolean {
+  return versionSpec.startsWith('lts');
+}
+
+function resolveLtsAliasFromManifest(
+  versionSpec: string,
+  stable: boolean,
+  manifest: INodeRelease[]
+): string {
+  const alias = versionSpec.split('lts/')[1]?.toLowerCase();
+
+  if (!alias) {
+    throw new Error(
+      `Unexpected LTS alias '${alias}' for Node version '${versionSpec}'`
```

Comment:
Personal thought:
Such line breaks caused by `"printWidth": 80`.
Increasing it to 100 would make code a little bit nicer.

CC @maxim-lobanov